core/src/test/scala/unit/kafka/server/ReplicaManagerTest.scala

```diff
@@ -1253,9 +1253,11 @@ class ReplicaManagerTest {
     val countDownLatch = new CountDownLatch(1)
 
     // Prepare the mocked components for the test
+    val props = new Properties()
+    props.put(KafkaConfig.ReplicaSelectorClassProp, "org.apache.kafka.common.replica.RackAwareReplicaSelector")
```

Comment:
Hum... It seems that my suggestion to update this test was perhaps not good when I see the diff. I misread the test. It might be better to keep it and add another one which is somewhat similar. Sorry for the confusion.

Reply:
Thanks a lot for your suggestion, I added a new unit test.